docs/api/naip_pretrained_weights.csv

```diff
@@ -1,2 +1,2 @@
 Weight,Channels,Source,Citation,License
-Swin_V2_B_Weights.NAIP_RGB_SATLAS,3,`link <https://github.com/allenai/satlas>`__,`link <https://arxiv.org/abs/2211.15660>`__,"Apache-2.0"
\ No newline at end of file
+Swin_V2_B_Weights.NAIP_RGB_SI_SATLAS,3,`link <https://github.com/allenai/satlas>`__,`link <https://arxiv.org/abs/2211.15660>`__,"ODC-BY"
```

Comment:
Thanks for correcting this license

Reply:
Wait, but https://huggingface.co/allenai/satlas-pretrain has an Apache-2.0 license?